devcluster/config.py

```diff
@@ -156,11 +156,12 @@ def build_stage(self, poll, logger, state_machine):
                 "kill_signal": "TERM",
                 "run_args": run_args,
                 "post": [
+                    {"logcheck": {"regex": "listening on IP"}},
```

Comment:
what about just `listening on`?  I think `listening on Unix socket` is also sufficient to meet these requirements.

Also, I think this happens to work but it introduces a race condition.  There's a place in `devcluster.py::StateMachine.advance_stage()` where the first postcommand before starting the process itself to ensure the logcheck can't miss any logs, but that's basically a hack that only works for the first logcheck.  After that, it's possible new log lines are added between when the first logcheck reports it is complete and before the second one is emitted.

That's pretty easy to fix.

Question: would it make more sense to you if the second log check started checking logs from the beginning of the process's lifetime?  Or should it always pick up on the first line after the previous log check has completed?

Reply:
ok, just `listening on IP` without the 2nd check seems to work. sometimes master connects before it’s ready to do stuff, but it is able to retry and recover from that error.
I think `listening on IP` makes more than `listening on` or `... unix socket` because we're connecting over tcp socket after all, not unix.

Reply:
let's go with that for now then.